packages/i18n/src/resources/en.json

```diff
@@ -133,7 +133,7 @@
   "com.affine.auth.toast.message.signed-in": "You have been signed in, start to sync your data with AFFiNE Cloud!",
   "com.affine.auth.toast.title.failed": "Unable to sign in",
   "com.affine.auth.toast.title.signed-in": "Signed in",
-  "com.affine.backButton": "Back Home",
+  "com.affine.backButton": "Back",
```

Comment:
`com.affine.backButton` also effect `404 Not Found` page, please to consider use a new key or give `Back Home` button inside `404` page a new key.

Reply:
I will add a new key called `com.affine.notFoundPage.backButton`. You don't need to take any action.